content/guides/models/track/environment-variables.md

```diff
@@ -36,45 +36,46 @@ os.environ["WANDB_SWEEP_ID"] = "b05fq58z"
-| **WANDB_RUN_GROUP**       | Specify the experiment name to automatically group runs together. See [grouping]({{< relref "/guides/models/track/runs/grouping.md" >}}) for more info.                                 |
-| **WANDB_RUN_ID**          | Set this to a globally unique string (per project) corresponding to a single run of your script. It must be no longer than 64 characters. All non-word characters will be converted to dashes. This can be used to resume an existing run in cases of failure.      |
-| **WANDB_SILENT**           | Set this to **true** to silence wandb log statements. If this is set all logs will be written to **WANDB_DIR**/debug.log               |
-| **WANDB_SHOW_RUN**        | Set this to **true** to automatically open a browser with the run url if your operating system supports it.        |
-| **WANDB_SWEEP_ID**        | Add sweep ID tracking to `Run` objects and related classes, and display in the UI.           |
-| **WANDB_TAGS**             | A comma separated list of tags to be applied to the run.                 |
-| **WANDB_USERNAME**         | The username of a member of your team associated with the run. This can be used along with a service account API key to enable attribution of automated runs to members of your team.               |
-| **WANDB_USER_EMAIL**      | The email of a member of your team associated with the run. This can be used along with a service account API key to enable attribution of automated runs to members of your team.            |
+| **WANDB_ANONYMOUS** | Set this to `allow`, `never`, or `must` to let users create anonymous runs with secret urls. |
+| **WANDB_API_KEY** | Sets the authentication key associated with your account. You can find your key on [your settings page](https://app.wandb.ai/settings).  This must be set if `wandb login` hasn't been run on the remote machine.               |
+| **WANDB_BASE_URL** | If you're using [wandb/local]({{< relref "/guides/hosting/" >}}) you should set this environment variable to `http://YOUR_IP:YOUR_PORT` |
+| **WANDB_CACHE_DIR** | This defaults to \~/.cache/wandb, you can override this location with this environment variable |
+| **WANDB_CONFIG_DIR** | This defaults to \~/.config/wandb, you can override this location with this environment variable |
+| **WANDB_CONFIG_PATHS** | Comma separated list of yaml files to load into wandb.config. See [config]({{< relref "./config.md#file-based-configs" >}}). |
+| **WANDB_CONSOLE** | Set this to "off" to disable stdout / stderr logging. This defaults to "on" in environments that support it. |
+| **WANDB_DATA_DIR** | This location is used for staging artifacts during upload. The default depends on your platform, as it uses `user_data_dir` from `platformdirs` python package |
```

Comment:
```suggestion
| **WANDB_DATA_DIR** | The location where staging artifacts are uploaded. The default location depends on your platform, because it uses the value of `user_data_dir` from the `platformdirs` Python package. |
```